### DISCUSSION TITLE
Missions never go to natural formations/abandoned outposts

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#14134] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Missions from outposts never go to natural formations or abandoned outposts.

This makes sense for specific missions like escort or transport, but not for a bunch of other missions like killing or salvage.
This kinda sucks, because usually the natural formations are towards the deeper regions, and if there aren't missions there it disincentivises going in the direction that progresses the campaign, just because there aren't missions there. 

### Reproduction steps

1. Load up a campaign
2. teleport to the deeper regions, to an outpost that neighbours natural formations/abandoned outposts and at least one outpost
3. unlock missions either from the npc or using the `unlockmissions` command
4. observe that none of the missions go to the natural formations/abandoned outposts

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by ubertpendragon** [comment]
Almost every mission including killing monsters goes from AnyOutpost to AnyOutpost so it prioritizes going to outposts

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/05ff2ba8-b2a6-4f23-b5f1-889eb20cba1b)

**Comment by Rokvach** [comment]
Fixed and tested, working correctly. Closing discussion.
